src/CodeCoverage.php

```diff
@@ -346,15 +346,36 @@ public function append(array $data, $id = null, bool $append = true, $linesToBeC
 
         $this->tests[$id] = ['size' => $size, 'status' => $status];
 
-        foreach ($data as $file => $lines) {
+        foreach ($data as $file => $fileData) {
             if (!$this->filter->isFile($file)) {
                 continue;
             }
 
-            foreach ($lines as $k => $v) {
-                if ($v === Driver::LINE_EXECUTED) {
-                    if (empty($this->data[$file][$k]) || !\in_array($id, $this->data[$file][$k])) {
-                        $this->data[$file][$k][] = $id;
+            foreach ($fileData['lines'] as $line => $lineCoverage) {
```

Comment:
Thank you for moving away from that `$k => $v` confusion. Useful variable names, woo!